mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestObjects.java

```diff
@@ -374,10 +374,9 @@ public Response getObjectVersions(@Parameter(example = "mir_mods_00004711") @Pat
         if (cachedResponse.isPresent()) {
             return cachedResponse.get();
         }
-        List<MCRMetadataVersion> versions = MCRXMLMetadataManager.instance().getVersionedMetaData(id)
-            .listVersions();
+        List<Long> versions = MCRXMLMetadataManager.instance().listRevisions(id);
         return Response.ok()
-            .entity(new GenericEntity<>(versions, TypeUtils.parameterize(List.class, MCRMetadataVersion.class)))
+            .entity(new GenericEntity<>(versions, TypeUtils.parameterize(List.class, Long.class)))
```

Comment:
@rsteph-de I was told by Jens to ping you about this because it affects the REST API.